src/native/minipal/cpufeatures.c

```diff
@@ -277,6 +277,28 @@ int minipal_getcpufeatures(void)
                                             {
                                                 result |= XArchIntrinsicConstants_AvxVnni;
                                             }
+
+                                            if ((cpuidInfo[CPUID_EDX] & (1 << 19)) != 0)                                // Avx10
+                                            {
+                                                __cpuidex(cpuidInfo, 0x00000024, 0x00000000);
+                                                const int versionMask = 0xFF; // [7:0]
+                                                if((cpuidInfo[CPUID_EBX] & versionMask) >= 1) // version higher than 1
+                                                {
+                                                    result |= XArchIntrinsicConstants_Avx10v1;
+                                                }
+
+                                                const int vector256Mask = (1 << 17);
```

Comment:
nit: None of the other checks use a `const int` like this, they inline the `(1 << 17)` into the if statement and then add a comment indicating what bit it was checking

Reply:
I can make it to the commonly used form, it looks unnecessary to define this way as the bit-shift form should be easy enough to read in this case.